community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -656,6 +668,12 @@ object projects:
     dependencies = List(scalatest, scalatestplusJunit, scalatestplusScalacheck)
   )
 
+  lazy val monocle = SbtCommunityProject(
+    project = "Monocle",
+    sbtTestCommand = "monocleJVM/test",
```

Comment:
```suggestion
    sbtTestCommand = "coreJVM/test; macrosJVM/test; testJVM/test",
```

`monocleJVM/test` won't work here, because it depends on the `genericJVM` and `refinedJVM` subprojects, which in turn have dependencies that are not part of the community build (`shapeless2` and `refined`, respectively)

I see that `refined` publishes for Scala 3, this could be added to the community build a a later date.